### PR TITLE
dbus-python update to 1.3.2

### PR DESCRIPTION
**Pull request by HouLiXieBuRou**
Topic Description
-----------------
- avahi: fix build and bump REL
    add a PATCH variable to reference python3 instead of python2
- a2jmidid: enable RISC-V,LOONGARCH64 build
- ladish: fix build for riscv64 loongarch64
- cadence: fix build and bump REL
    autobuild4 requires packages with built-in makefiles to provide build scripts
- cinnamon: bump REL and update patch location
- xboxdrv: fix build and bump REL
    Added several patches.
    0000: fix xboxdrv.1 errors reported by lintian
    Fix all hyphen-used-as-minus-sign and spelling-error-in-manpage reports.
    0001: fix handling *flags environment variables containing >1 flags
    0002: use python3 print()
    0003: port xboxdrvctl to python3
    0004: scons 4.2.0 no longer has env.has_key()
    Check env as an array.
    0005: use the new input_event 64-bit-compliant api

Package(s) Affected
-------------------

Security Update?
----------------

No

Build Order
-----------

```
#buildit dbus-python avahi a2jmidid zita-ajbridge ladish cadence cinnamon xboxdrv
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Comment by HouLiXieBuRou** [comment]
amd64
avahi
image
ladish
image
image
cadence
image
xboxdrv
image
a2jmidid
image
zita....
image

File: app-multimedia/cadence/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+MAKE_DEF=(
+	PREFIX='/usr'
+)
+MAKE_AFTER=(
+	CFLAGS="$CFLAGS"
+	LDFLAGS="$LDFLAGS"
+)
+
```
Comment (Cyanoxygen) [inline app-multimedia/cadence/autobuild/build:8]: Please add informative output in this script.

File: app-network/avahi/autobuild/prepare [hunk 2]
```diff
@@ -1,2 +1,3 @@
 wget https://github.com/lathiat/avahi/raw/master/service-type-database/build-db \
```
Comment (Cyanoxygen) [inline app-network/avahi/autobuild/prepare]: This download must be handled by ACBS.

File: app-network/avahi/autobuild/prepare [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
     -O "$SRCDIR"/service-type-database/build-db
+export PYTHON=/usr/bin/python3
```
Comment (Cyanoxygen) [inline app-network/avahi/autobuild/prepare:2]: Please add extra output for this.

**Review by Cyanoxygen — CHANGES_REQUESTED** [review]

**Review by Cyanoxygen — APPROVED** [review]

**Pull request merged**